dev-docs/bidder-adaptor.md

```diff
@@ -495,9 +495,102 @@ if (bid.mediaType === 'video' || (videoMediaType && context !== 'outstream')) {
 
 #### Long-Form Video Content
 
-To support long-form videos it is the responsibility of the adapter to convert their categories into [IAB accepted subcategories]( http://iabtechlab.com/wp-content/uploads/2017/11/IAB_Tech_Lab_Content_Taxonomy_V2_Final_2017-11.xlsx) (links to MS Excel file). Each bid request must return one IAB subcategory.
+{: .alert.alert-info :}
+Following is Prebid's way to setup bid request for long-form. Apadters are free to choose their own approach.
+
+Prebid now accepts multiple bid responses for a single `bidRequest.bids` object. For each Ad pod Prebid expects you to send back n bid responses. It is up to you how you send back bid responses. Prebid's recommendation is that you expand an adpod placement into a set of request objects according to total adpod duration and the range of duration seconds. It also depends on your endpoint as well how you may want to create your request for long-form. Appnexus adapter follows below algorithm to expand its placement. 
```

Comment:
consistency with `Ad pod` vs `adpod`. I think we should just use `adpod`.

Reply:
`send back n bid responses...` capitalize and italicized please. Thanks

Reply:
When i want to explain code, i have used `adpod` context. 
If not in code I have used `Ad pod` as per usage on IAB website https://video-guide.iab.com/video-ad-tech-overview